Remotion/Web/Web.Development.WebTesting.IntegrationTests.RequireUI/DiagnosticInformationCollectioningRequestErrorDetectionStrategy.cs

```diff
@@ -0,0 +1,83 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using Coypu;
+using Remotion.Web.Development.WebTesting.RequestErrorDetectionStrategies;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.RequireUI
+{
+  /// <summary>
+  /// Testclass to be able to seperate Tests for <see cref="IRequestErrorDetectionStrategy"/> implementation and calls.
+  /// </summary>
+  public class DiagnosticInformationCollectioningRequestErrorDetectionStrategy : IRequestErrorDetectionStrategy
```

Comment:
Duplicating this is not ideal since it is quite a bit of code. We do have an infrastructure project into which we could move this class. This way, we don't have to duplicate the strategy. Calling @MichaelKetting to confirm that this is a good approach :D